java/client/src/main/java/glide/api/commands/servermodules/Json.java

```diff
@@ -0,0 +1,456 @@
+     * @param key The <code>key</code> of the JSON document.
+     * @param paths List of paths within the JSON document.
+     * @param options Options for formatting the byte representation of the JSON data. See <code>
+     *     JsonGetOptions</code>.
+     * @return
+     *     <ul>
+     *       <li>If one path is given:
+     *           <ul>
+     *             <li>For JSONPath (path starts with <code>$</code>): Returns a stringified JSON list
+     *                 replies for every possible path, or a string representation of an empty array,
+     *                 if path doesn't exist. If <code>key</code> doesn't exist, returns None.
+     *             <li>For legacy path (path doesn't start with <code>$</code>): Returns a string
+     *                 representation of the value in <code>paths</code>. If <code>paths</code>
+     *                 doesn't exist, an error is raised. If <code>key</code> doesn't exist, returns
+     *                 None.
+     *           </ul>
+     *       <li>If multiple paths are given: Returns a stringified JSON, in which each path is a key,
+     *           and it's corresponding value, is the value as if the path was executed in the command
+     *           as a single path.
+     *     </ul>
+     *     In case of multiple paths, and <code>paths</code> are a mix of both JSONPath and legacy
+     *     path, the command behaves as if all are JSONPath paths.
+     * @example
+     *     <pre>{@code
+     * JsonGetOptions options = JsonGetOptions.builder()
+     *                              .indent("  ")
+     *                              .space(" ")
+     *                              .newline("\n")
+     *                              .build();
+     * GlideString value = client.Json.get(client, gs("doc"), new GlideString[] {gs("$.a"), gs("$.b")}, options).get();
+     * assert value.equals(gs("{\n \"$.a\": [\n  1.0\n ],\n \"$.b\": [\n  2\n ]\n}"));
+     * }</pre>
+     */
+    public static CompletableFuture<GlideString> get(
+            @NonNull BaseClient client,
+            @NonNull GlideString key,
+            @NonNull GlideString[] paths,
+            @NonNull JsonGetOptionsBinary options) {
+        return executeCommand(
+                client,
+                new ArgsBuilder().add(gs(JSON_GET)).add(key).add(options.toArgs()).add(paths).toArray());
+    }
+
+    /**
+     * A wrapper for custom command API.
+     *
+     * @param client The client to execute the command.
+     * @param args The command line.
+     */
+    private static <T> CompletableFuture<T> executeCommand(BaseClient client, String[] args) {
+        return executeCommand(client, args, false);
+    }
+
+    /**
+     * A wrapper for custom command API.
+     *
+     * @param client The client to execute the command.
+     * @param args The command line.
+     * @param returnsMap - true if command returns a map
+     */
+    @SuppressWarnings({"unchecked", "SameParameterValue"})
```

Comment:
Why you need to suppress second warning

Reply:
It's complaining that `boolean returnsMap` is always false. But I do think we need to keep this option for future commands.